cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -535,6 +535,31 @@
         /var/snap/{,*/} r,
         /var/snap/*/**/ rw,
 
+        # Allow creating placeholder directory in /tmp/.snap/ as support for
+        # the writable mimic code that can poke holes in arbitrary read-only
+        # places using tmpfs and bind mounts.
+        /tmp/ r,
+        /tmp/.snap/ rw,
+        /tmp/.snap/** rw,
```

Comment:
Not a blocker: I would probaly rewrite this is:
```
/tmp/.snap/{,**} rw,
```

Reply:
Applied